static/js/vue-cdr-access/src/components/full_record/restrictedContent.vue

```diff
@@ -20,6 +20,9 @@
                         <i class="fa fa-search" aria-hidden="true"></i> View</a>
                 </div>
             </template>
+            <template v-if="hasPermission(recordData, 'viewHidden')">
```

Comment:
We should add tests to verify that the single-use-link component shows up with viewHidden permissions, and not without them